.github/workflows/Dockerfile.tools

```
@@ -0,0 +1,3 @@
+FROM registry.gitlab.com/openwrt/buildbot/buildworker-3.4.1
+
+COPY --chown=buildbot:buildbot tools.tar /tools.tar
```

```suggestion
COPY --chown=buildbot:buildbot tools.tar /whereever/
```

Why would you want to have this shipped as a tar and not unpacked?

---

I decided to take the tar route to be safe on link handling and other timestamps.

On the kernel.yaml workflow we don't link but we extract them.

If you want I can do one more test with the copy and link but currently the extraction and make tools is less than a minute.

---

Right, since we don't store all files but only the tools in there!